addons/project/models/project_task.py

```diff
@@ -1797,10 +1798,10 @@ def action_dependent_tasks(self):
         }
         if self.dependent_tasks_count == 1:
             action['view_mode'] = 'form'
-            action['res_id'] = self.dependent_ids.id
+            action['res_id'] = self.dependent_ids.filtered(lambda t: t.state not in CLOSED_STATES).id
```

Comment:
See with @aju-odoo ,

Why not letting as it was before to see all blocking tasks? Since in the sublist view we displayed all tasks even the ones closed? 🧐 

If we really want that then we should instead change the compute of `dependent_tasks_count` to only have the open blocking tasks, otherwise with your current changes, if the task has a blocking tasks but that blocking one is closed then the form view displayed will be empty (creation mode) instead of seeing the form view of the blocking task. 😕

Reply:
but i already changed the compute
![image](https://github.com/odoo/odoo/assets/64937092/65dbb736-6fd7-4701-b2e1-e78de9694efd)
am i missing something ?

Reply:
Oh sorry. 🙈 

I probably check the code without "checkouting" your branch. 😬